rir/src/interpreter/interp.cpp

```diff
@@ -2816,6 +2900,9 @@ SEXP evalRirCode(Code* c, InterpreterInstance* ctx, SEXP* env,
             c->registerInvocation();
             assert(c->code() <= pc && pc < c->endCode());
             SEXP e = ostack_pop(ctx);
+            if (LazyEnvironment::check(e)) {
+                e = (LazyEnvironment::unpack(e))->create();
+            }
```

Comment:
why don't we need to update the stack here?

Reply:
We need and we are doing it.